xrpl-std/src/lib.rs

```diff
@@ -123,6 +126,16 @@ pub fn account_keylet(aid: &AccountID) -> Option<Keylet> {
 
 pub fn update_current_escrow_data(data: ContractData) {
     unsafe {
-        host_lib::update_data(data.as_ptr(), data.len());
+        host::update_data(data.as_ptr(), data.len());
     }
 }
+
+/// This function is called on panic, but only in the WASM architecture. In non-WASM (e.g., in the
+/// Host Simulator) the standard lib is available, which includes a panic handler.
+#[cfg(target_arch = "wasm32")]
+#[panic_handler]
+fn panic(_info: &::core::panic::PanicInfo) -> ! {
```

Comment:
In my build, this is required for builds that target `wasm32-unknown-unknown`, FWIW.